config/crd/bases/operator.containers.carbonblack.io_cbcontainersagents.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
please delete your lcoal
`bin/controller-gen` & /bin_old/controller-gen` files before running the make commands.
you can remove the all bin folder